README.md

```diff
@@ -75,16 +94,43 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
-
+(Had to refer to past lessons/exercises)
+
+app.get("/", function(req, res){
+  res.render("checkpoints");
+  console.log("Get/Read");
+});
+
+app.post("/checkpoints", function(req, res){
+  Checkpoint.create(req.body.checkpoints).then(function(checkpoint){
+    res.redirect("/checkpoints/" + checkpoint.name);
+    console.log("Post/Create");
+  });
+});
+
+app.post("/checkpoints/:name", function(req, res){
+  Checkpoint.findOneAndUpdate({name: req.params.name}, req.body.checkpoint, {new: true}).then(function(checkpoint){
+    res.redirect("/checkpoints/" + checkpoint.name);
+    console.log("Update/Edit");
+  });
+});
+
+app.post("/checkpoints/:name/delete", function(req, res){
+  Checkpoint.findOneAndRemove({name: req.params.name}).then(function(){
+    res.redirect("/checkpoints")
+    console.log("Delete/Destroy");
+  });
+});
 ```
 
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-Your answer...
+Express is a backend framework based on Sinatra and uses JavaScript as its' core language while Rail's core language is Ruby. This makes express easier to deal with since you can use the same language throughout, causing less headaches since the syntax will be the same throughout your whole program.
+
+Rails is also an SQL database while Express is a NoSQL database
```

Comment:
Rails or Express is agnostic of databases, it's just that Mongo got popular with Express for the whole stack in one language thing